client/packages/invoices/src/Prescriptions/DetailView/Toolbar.tsx

```diff
@@ -125,7 +125,11 @@ export const Toolbar: FC = () => {
                   defaultValue={new Date()}
                   value={DateUtils.getDateOrNull(prescriptionDate)}
                   format="P"
-                  onChange={handleDateChange}
+                  // Using onAccept rather than onChange -- on mobile, onChange
+                  // is triggered when first opening the picker, which causes UI
+                  // conflict with the confirmation modal
+                  onAccept={handleDateChange}
+                  onChange={() => {}}
```

Comment:
`onChange` is a required prop, so passing a dummy function seemed simpler than refactoring DatePicker to handle non-compulsory `onChange`.